docs/content/doc/usage/backup-and-restore.en-us.md

```diff
@@ -57,7 +57,7 @@ The command has to be executed with the `RUN_USER = <OS_USERNAME>` specified in
 Example:
 
 ```none
-docker exec -u <OS_USERNAME> -it -w <--tempdir> $(docker ps -qf "name=<NAME_OF_DOCKER_CONTAINER>") bash -c '/app/gitea/gitea dump -c </path/to/app.ini>'
+docker exec -u <OS_USERNAME> -it -w <--tempdir> $(docker ps -qf name=^<NAME_OF_DOCKER_CONTAINER>$) bash -c '/app/gitea/gitea dump -c </path/to/app.ini>'
```

Comment:
are you sure that name argument does not still need to be in quotes?

Reply:
Yes I'm sure.

Reply:
This is shell dependent. If you are running the zsh the ^ and $ will be picked up an interpreted by the shell.

Reply:
We use this with `bash`, but I agree.. this is probably true.

Reply:
So could you stick it in quotes as suggested then I will be happy to approve.